lib/cli.rs

```diff
@@ -30,14 +30,21 @@ mod parse;
 mod payment_str_params;
 mod session_str_params;
 mod simple_args;
+#[cfg(feature = "sdk")]
```

Comment:
I worry that making these public in the client, rather than re-implementing them in the SDK could create external dependencies on this code. This would cause any changes to these items to be breaking and cause it to be a headache for future maintenance of the client. Perhaps these items can just be copied over to the SDK itself.

Reply:
I didn't re implement methods, every rpc call, deploys everything is relaying on the casper client. The purpose is to have the client served as a library, not to duplicate code to have double maintenance, already few methods are exposed, I don't see why 2 more would be problematic. If the casper client/node evolve, then the sdk is following.